src/http/api/routes/files.js

```diff
@@ -2,7 +2,66 @@
 
 const resources = require('./../resources')
 const mfs = require('ipfs-mfs/http')
+const fs = require('fs')
+const path = require('path')
+const tempy = require('tempy')
+const multipart = require('ipfs-multipart')
+const pull = require('pull-stream')
+const toPull = require('stream-to-pull-stream')
+const pushable = require('pull-pushable')
 
+const streams = []
+const filesDir = tempy.directory()
```

Comment:
We should remove temporary files after successful upload, but also have a plan for what happens with temp files produced by failed/aborted  ones. I don't think we can assume temp dir will be cleared by the OS. 
Leaving them forever may be a self-administered DoS attack (eventually running out of disk space)

Reply:
yes definitely should go like this
- success adding ? delete file
- errors during session ? define a way to remove only the last chunk from the file ( or have one file per chunk ) this would allow for resumable add's
- on new session start async check the temp folder and delete all files older than 1 hour
- also we need to handle open streams, too many can become a problem